framework/src/media/audio/audio_manager.c

```diff
@@ -678,7 +678,7 @@ audio_manager_result_t set_audio_stream_in(uint8_t channels, uint32_t sample_rat
 
 	medvdbg("[IN] Device samplerate: %u, User requested: %u\n", config.rate, sample_rate);
 
-	medvdbg("actual input card id = %d\n", g_actual_audio_out_card_id);
+	medvdbg("actual input card id = %d\n", g_actual_audio_in_card_id);
```

Comment:
please add this in next PR =) sorry for my mistake =D

Reply:
Okay